packaging/nnstreamer.spec

```diff
@@ -35,6 +35,7 @@
 %define		lua_support 1
 %define		tvm_support 1
 %define		snpe_support 1
+%define		trix_engine_support 1
```

Comment:
Set it 0 if it's 
- DA build (LINE 110)
- Tizen 6.5 or lower (around LINE 43-70)

Reply:
Updated, thanks.